backend/package.json

```diff
@@ -32,6 +32,7 @@
         "body-parser": "^1.19.1",
         "dotenv": "^10.0.0",
         "express": "^4.17.1",
-        "jest": "^27.4.5"
+        "jest": "^27.4.5",
+        "tsconfig-paths": "^3.12.0"
```

Comment:
Why did you use "tsconfig-paths" lib? Absolute paths are already working with 'baseUrl' setting
Please remove it

Reply:
I dont fix this file - its merge from develop branch before